### DISCUSSION TITLE
Future Accessability via TypeDoc?

### DISCUSSION THREAD

**Discussion by milk-and-mocha** [squidfunk/mkdocs-material#7626] [Ideas]
Currently I have a mkdocs project that is using both mkapi and mkdocs-typedoc plugins, however, the mkdocs-material theme installed does not work with the mkdocs-typedoc plugin. So now I have a site that has a consistent theme except for one section.

**Comment by alexvoss** [comment]
As far as I can tell, typedoc produces ready-made HTML, so it is not surprising that this does not integrate. I would say that it is the mkdocs-typescript plugin that does not integrate with MkDocs (any theme) rather than the other way round. The Insiders edition of mkdocstrings [has support for TypeScript](https://mkdocstrings.github.io/typescript/). That might give you what you are looking for.

**Reply by kamilkrzyskow** [reply]
Never used TypeDoc, but here it says there is JSON output support:
- https://typedoc.org/options/output/#json

So `mkdocs-typedoc` plugin should use this option to get a JSON file output and then it should create MkDocs Page objects, which would use the Material for MkDocs theme. Something of sort like the Blog plugin generates Category and Archive pages.
